home/management/commands/export_csv_content.py

```diff
@@ -31,29 +41,53 @@ def get_whatsapp_messages(page):
             wa_msgs = []
             for msg in page.whatsapp_body._raw_data:
                 wa_msgs.append(msg["value"]["message"])
-            return wa_msgs
+            return format_messages(wa_msgs)
 
-        def get_tags(page):
-            tags = []
-            for tag in ContentPageTag.objects.filter(content_object_id=page.id):
-                tags.append(tag.tag.name)
-            return tags
+        def get_messenger_messages(page):
+            messenger_msgs = []
+            for msg in page.messenger_body._raw_data:
+                messenger_msgs.append(msg["value"]["message"])
+            return format_messages(messenger_msgs)
+
+        def get_viber_messages(page):
+            viber_msgs = []
+            for msg in page.viber_body._raw_data:
+                viber_msgs.append(msg["value"]["message"])
+            return format_messages(viber_msgs)
 
         def get_parent_page(page):
             if not HomePage.objects.filter(id=page.get_parent().id).exists():
                 return page.get_parent().title
 
+        def format_messages(message_list):
+            message_delimiter = "\n\n\n"
+            return message_delimiter.join(message_list)
+
+        def format_list_from_query(unformatted_query):
+            list_delimiter = ", "
+            return list_delimiter.join(str(x) for x in unformatted_query)
+
+        def get_rows(page):
+            dict_row = {}
+            dict_row["parent"] = get_parent_page(page)
+            dict_row["web_title"] = page.title
+            dict_row["web_subtitle"] = page.subtitle
+            dict_row["web_body"] = page.body
+            dict_row["whatsapp_title"] = page.whatsapp_title
+            dict_row["whatsapp_body"] = get_whatsapp_messages(page)
+            dict_row["messenger_title"] = page.messenger_title
+            dict_row["messenger_body"] = get_messenger_messages(page)
+            dict_row["viber_title"] = page.viber_title
+            dict_row["viber_body"] = get_viber_messages(page)
+            dict_row["translation_tag"] = str(page.translation_key)
+            dict_row["tags"] = format_list_from_query(page.tags.all())
+            dict_row["quick_replies"] = format_list_from_query(page.quick_replies.all())
+            dict_row["triggers"] = format_list_from_query(page.triggers.all())
+            dict_row["locale"] = page.locale
+            return dict_row
+
         rows = []
-        for page in ContentPage.objects.all():
-            row = {
-                "web_title": page.title,
-                "web_subtitle": page.subtitle,
-                "web_body": page.body,
-                "whatsapp_title": page.whatsapp_title,
-                "whatsapp_body": "\n".join(get_whatsapp_messages(page)),
-                "tags": ",".join(get_tags(page)),
-                "parent": get_parent_page(page),
-            }
-            rows.append(row)
+        for page in ContentPage.objects.filter(id=1194).all():
```

Comment:
why 1194?

Reply:
whoops, that was for testing, I thought I had taken it out